features/polyExplorer/src/components/clusterStory/clusterStory.jsx

```diff
@@ -4,6 +4,7 @@ import Screen from "../screen/screen.jsx";
 import "./clusterStory.css";
 
 const defaultProgressBarColor = "#3BA6FF";
+const fadingTopBackgroundDefaultDistance = "600px";
 
 /* This component is used as a base for the different Cluster Stories screens. In order to style the screen,
     some classes with CSS properties already implemented have been created. You just need to add those classNames 
```

Comment:
Since you have added the new variable `--story-color-primary`, there is a text fragment that we don't need anymore in the comment that I made (from the middle of line 14 to 18):
```
You will also need to add the "color" attribute
        to its related CSS file. For example, in messengerStory.css:
                .messenger-story .cluster-story-title {
                    color: var(--color-primary-messenger-story);
                }
```